docker/archive/dest.go

```diff
@@ -19,17 +19,32 @@ func newImageDestination(ctx *types.SystemContext, ref archiveReference) (types.
 	if ref.destinationRef == nil {
 		return nil, errors.Errorf("docker-archive: destination reference not supplied (must be of form <path>:<reference:tag>)")
 	}
-	fh, err := os.OpenFile(ref.path, os.O_WRONLY|os.O_EXCL|os.O_CREATE, 0644)
+	// tries to open a file first to accomodate for load when using stdout
+	// as the file is created first then written to
+	// if the file doesn't exist then it creates a new file for exclusive use
+	// if the file exists, it checks to ensure the size is 0
```

Comment:
“to accomodate for load” is difficult to understand for me as a non-native speaker.

(In general, don’t write comments describing what every line does; that should be obvious from the code, and such comments are over time more likely to become obsolete and misleading than to be helpful; instead, write comments saying what is the code trying to do, most importantly the situation/problem (“Note that this needs to account for $this_weird_case”), and, if non-obvious, the solution.)